src/main/java/servlets/OAuthConsentServlet.java

```diff
@@ -43,9 +43,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Get the current session or create a new one.
     HttpSession session = request.getSession(true);
     session.setAttribute(OAuthConstants.SHEETS_SESSION_KEY, state);
-
-    // Redirect user to OAuthConsentPage.
-    response.sendRedirect(getOAuthRedirectURL(state));
+    try {
+      String redirectUrl = getOAuthRedirectURL(state);
+      response.sendRedirect(redirectUrl);
+    } catch(NullPointerException e) {
+      response.sendError(400, "The domain variable has not been set");
```

Comment:
Might be nice to have a more user-friendly error message here. This one seems a bit cryptic